.github/workflows/reusable-upload.yaml

```
@@ -48,36 +56,22 @@ on:
         required: true
 
 jobs:
-  build-and-upload:
-    runs-on: blacksmith-8vcpu-ubuntu-2204
+  upload:
+    name: upload ${{ inputs.package-git-tag }}
```

```suggestion
    name: Upload ${{ inputs.package-git-tag }}
```
add contract name as well?